spec/spec_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'dotenv'
+Dotenv.load('.secrets', '.env.test', '.env')
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Is there anything within this test / set of tests that actually needs anything in `.env.test`? Or even `.env`? I'm guessing you have the `.load` method here because it's needed for RSpec to actually get the environment variables that you do need? Just wondering if there's an alternative to Dotenv.

Reply:
The ones I need are the HTTP basic auth username and password (currently residing in .secrets) and YUL_DC_SERVER (which I made up and could live anywhere, I put it in .env.test when I was running these).  And yeah, I used Dotenv.load to get hold of those.  Quite happy to use other methods but not sure what/how.

Reply:
pulled the gem out and just exported the values manually for my local work.